src/App.test.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] `required`
- [ ] The use of linters helps to diagnose and fix technical issues—e.g. code smells—in the code. As a result, fewer defects make their way to production. Readable, maintainable, and more consistent code. Linters can help teams achieve a more readable and consistent style, through the enforcement of its rules.
- [ ] Customizing the Linters by `Disabling the eslint` defeats the purpose of using it please enable it.

Reply:
@mwanawabangona I understand what your saying but I am using  /* eslint-disable no-unused-vars */ because that is for importing react if I remove react I get a different lintor error so there is nothing I can do.